### PR TITLE
Add recipe for spyder-reports.

### PR DESCRIPTION
**Pull request by rlaverde**
Spyder-IDE plugin for Markdown reports using Pweave.

ping @ccordoba12 @goanpeca

### CODE REVIEW

File: recipes/spyder-reports/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - spyder_reports
+
+about:
+  home: https://github.com/spyder-ide/spyder-reports
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (goanpeca) [inline recipes/spyder-reports/meta.yaml]: LICENSE.txt ?

Comment (rlaverde) [inline recipes/spyder-reports/meta.yaml]: yes, nice catch 🙈

**Comment by ccordoba12** [comment]
+1 from me to be maintainer.

**Comment by ccordoba12** [comment]
This release is not compatible with Python 2, so you need to skip it.

**Comment by rlaverde** [comment]
This is failing because conda-foprge Pweave doesn't support py3.6
I already open a PR against Pweave feedstock https://github.com/conda-forge/pweave-feedstock/pull/1

**Comment by ccordoba12** [comment]
@rlaverde, you need to readd the recipe here in staged-recipes to fix a conda-forge token issue.

Please take a look at https://github.com/conda-forge/staged-recipes/pull/3537 to see how to do it.

**Comment by rlaverde** [comment]
I think this is ready 😄

**Comment by goanpeca** [comment]
@jakirkham  :-p

**Comment by jakirkham** [comment]
Sorry I'm kinda swamped ATM. Am going to hand this off to @conda-forge/staged-recipes to take a look. Hope that is ok.

**Comment by goanpeca** [comment]
@jakirkham sure, no prob :-p

**Comment by mwcraig** [comment]
Looks good, merging...

**Comment by ccordoba12** [comment]
Thanks Matt!

**Pull request merged**